crates/rattler_installs_packages/src/index/package_database.rs

```diff
@@ -232,7 +233,35 @@ impl PackageDb {
     ) -> miette::Result<Wheel> {
         // TODO: add support for this currently there are not saved
```

Comment:
Remove this todo?

Reply:
No, currently we do not cache our direct source dependencies

Reply:
Oh right! For repeated runs you mean :)

Reply:
```suggestion
        // TODO: add support for this currently there are not cached, they will be repeatedly downloaded between runs
```